anta/catalog.py

```diff
@@ -35,8 +35,37 @@
 
 logger = logging.getLogger(__name__)
 
-# { <module_name> : [ { <test_class_name>: <input_as_dict_or_None> }, ... ] }
-RawCatalogInput = dict[str, list[dict[str, Optional[dict[str, Any]]]]]
```

Comment:
Removing this will break AVD. We use that type here: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py#L170